mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/GhostMailbox.java

```diff
@@ -35,6 +35,14 @@
     public static final String TYPE = "type";
     public static final String MAILBOX_ID = "mailboxId";
 
+    public static boolean isDebugEnabled() {
+        return LOGGER.isDebugEnabled();
+    }
+
+    public static boolean isInfoEnabled() {
+        return LOGGER.isDebugEnabled();
```

Comment:
You return here if debug is enabled, nothing to do with info...

I don't think the method `isInfoEnabled` would be used anyway, and it's not in your code for now. We can remove it?